### PR TITLE
Add dependabot definition

### PR DESCRIPTION
**Pull request by jortegac**
Dependabot raises pull requests to update dependencies. Depending on how your repository is configured, Dependabot may raise pull requests for version updates and/or for security updates. You manage these pull requests in the same way as any other pull request, but there are also some extra commands available. For information about enabling Dependabot dependency updates, see "[Configuring Dependabot security updates](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates)" and "[Configuring Dependabot version updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates)."

TL;DR: 
 - Visit your setting https://github.com/assafelovic/gpt-researcher/settings/security_analysis
 - Enable Dependabot

### CODE REVIEW

**Comment by jortegac** [comment]
Closing cuz it looks like I pushed the changes with the other branch 😅
